test/Game/gameMinter/gameMinter.test.ts

```diff
@@ -20,22 +20,82 @@ import {sendMetaTx} from '../../sendMetaTx';
 
```

Comment:
you added this empty file test/Game/game.ts
(see above)

Reply:
it is an oversight too